tests/test_http.sh

```diff
@@ -24,6 +24,20 @@ rlJournalStart
         assert_up_and_running
     rlPhaseEnd
 
+    rlPhaseStartTest "Check for SSL warning message"
+        rlRun -t -c "docker exec -i kiwi_web /Kiwi/manage.py createsuperuser \
+            --username testadmin --email testadmin@domain.com --noinput"
+        rlRun -t -c "docker exec -i kiwi_web /Kiwi/manage.py set_fake_passwords"
```

Comment:
`STDERR: CommandError: Only available in debug mode`, see
https://github.com/kiwitcms/enterprise/blob/master/.github/workflows/testing.yml#L146